src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -146,7 +146,7 @@ private EpisodeDto MapEpisodeToEpisodeDto(BssEpisode episode)
             EpisodeType = episode.episode_type,
             ScreeningName = "Breast Screening",
             NhsNumber = episode.nhs_number,
-            EpisodeOpenDate = episode.episode_date,
+            EpisodeOpenDate = string.IsNullOrEmpty(episode.episode_date) ? null : DateOnly.FromDateTime(DateTime.ParseExact(episode.episode_date, new[] { "dd-MM-yyyy", "dd/MM/yyyy" }, CultureInfo.InvariantCulture, DateTimeStyles.None)),
```

Comment:
I think we should extract this logic to a helper method and also use it for FirstOfferedAppointmentDate, ActualScreeningDate and EarlyRecallDate